unidist/core/backends/mpi/core/controller/api.py

```diff
@@ -49,6 +51,56 @@
 topology = dict()
 # The global variable is responsible for if MPI backend has already been initialized
 is_mpi_initialized = False
+threads = []
+BACKOFF = 0.001
```

Comment:
We should add a config and get the value from it. https://github.com/modin-project/unidist/blob/master/unidist/config/backends/mpi/envvars.py